hs/app/reach/Main.hs

```diff
@@ -2484,12 +2502,16 @@ support = command "support" $ info (pure g) d
         $ setRequestBodyJSON (object x)
       <$> parseRequest ("POST " <> u)
       >>= httpLBS
-    g = liftIO $ do
+    doDefaultBehavior = liftIO $ do
```

Comment:
I think the default behavior should be `useArgs [ "index.rsh", "index.mjs" ]` and you should change `go` to call a new `useArgs` function after giving it the arguments